pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+7. Security Implications - If there are security concerns in relation to the PHEP, those concerns should be explicitly written out to make sure reviewers of the PHEP are aware of them.
+8. How to Teach This - For a PHEP that adds new functionality or changes behavior, it is helpful to include a section on how to teach users, new and experienced, how to apply the PHEP to their work.
+
+    This section may include key points and recommended documentation changes that would help users adopt a new feature or migrate their code to use a language change.
+
+9. Reference Implementation - The reference implementation must be completed before any PHEP is given status "Final", but it need not be completed before the PHEP is accepted. While there is merit to the approach of reaching consensus on the specification and rationale before writing code, the principle of "rough consensus and running code" is still useful when it comes to resolving many discussions of API details.
+
+    The final implementation must include test code and documentation appropriate for the relevant PyHC project(s).
+
+10. Rejected Ideas - Throughout the discussion of a PHEP, various ideas will be proposed which are not accepted. Those rejected ideas should be recorded along with the reasoning as to why they were rejected. This both helps record the thought process behind the final version of the PHEP as well as preventing people from bringing up the same rejected idea again in subsequent discussions.
+
+    In a way this section can be thought of as a breakout section of the Rationale section that is focused specifically on why certain ideas were not ultimately pursued.
+
+11. Open Issues - While a PHEP is in draft, ideas can come up which warrant further discussion. Those ideas should be recorded so people know that they are being thought about but do not have a concrete resolution. This helps make sure all issues required for the PHEP to be ready for consideration are complete and reduces people duplicating prior discussion.
+12. Footnotes - A collection of footnotes cited in the PHEP, and a place to list non-inline hyperlink targets.
+13. Copyright/license - Each new PHEP must be placed under a dual license of public domain and [CC0-1.0-Universal](https://choosealicense.com/licenses/cc0-1.0/) (see this PHEP for an example).
+
+# PHEP Formats and Templates
+PHEPs are UTF-8 encoded text files using the Markdown format. Markdown allows for rich markup that is still quite easy to read, but also results in good-looking and functional HTML. A future PHEP may include a template. PHEP Markdown must be compatible with the most recent version of the [CommonMark specification](https://spec.commonmark.org/) at the time the PHEP is approved. PHEPs must not use [GitHub extensions](https://github.github.com/gfm/).
+
+PHEPs should be restricted to the ASCII subset of UTF-8 where practicable. Examples of cases where it is not practicable include personal names.
```

Comment:
@namurphy let's have this arm-wrestling here :) My rationale is that you can quickly wind up down a very deep hole of hypen vs. en-dash vs. em-dash, open/close quotes vs. straight quotes, etc. etc. and enforcing consistency even within a single document can be very hard. It's somewhat counter to the ideal of markdown as something that can be whipped out in a very simple editor without doing stacks of deadkey combinations.

I can be convinced (I think).

Reply:
I personally find em-dashes to help with readability, but I'll leave it totally up to you!  Please feel completely free to choose whichever way you prefer.

Reply:
I think I'm helping @namurphy push your arm down here, @jtniehof. Haha I am also very much in favor of en-dashes and em-dashes (when someone even knows what they are, I tend to instantly like that person more 😅). 

I think we're using modern enough technology that the distinction between UTF-8 and the ASCII subset isn't something people have to think about in practice. Hell, if we did enforce this rule, I'd end up having to reference ASCII charts to make sure I'm not using illegal characters, and that feels needlessly cumbersome. 

I get that it's surprisingly challenging just to keep things like straight quotes consistent among one document, and it does make editing/proofreading more complex. But I do have extensive professional experience doing that kind of editing from a previous life. I wouldn't hate being the guy who proofreads all PHEPs for consistency in things like this.

Reply:
(Plus, thinking out loud, it would take five seconds to implement an `enforce_straight_quotes(doc)` function with GPT-4, so that kind of proofreading can be automated...)

Reply:
Proposed solution, which will be in the next push:
-I'll remove the ASCII requirement
-Adding "The editors are responsible for enforcing PHEP format and consistency of style within a PHEP. They may use automated tools (e.g. pre-commit hooks) to assist."

There are six hundred little bits of style that really don't matter between PHEPs, and it seems to me the job of the editor to make sure things like quotes are matched. If the author of one PHEP wants to use strict ASCII, they should be allowed--if another wants to be a little more loose, they should be allowed. If somebody jams an entire PHEP full of emoji, then the editors and/or the community review process can decide if that makes sense.

Reply:
Oh, I should note CommonMark specifically says Unicode with no requirements on encoding.

Reply:
See new wording at https://github.com/jtniehof/standards/blob/4af2b621485d0ebb7ef4fb5aafb09544faae440a/pheps/phep-0001.md?plain=1#L182

Reply:
Going to resolve this. Documented as a rejected idea so a new thread can be opened up there if necessary.